frontend/src/app/initial-configuration/initial-configuration.component.ts

```diff
@@ -45,11 +45,13 @@ export class InitialConfigurationComponent implements OnInit {
   ngOnInit(): void {
     this.numberOfNodes = this.parametersService.getAllNodes();
```

Comment:
trzeba tu dodać setNode() przed tym getem bo jak teraz zaczynamy z wartoscia 5 w innych nodeach to trzeba to najpierw przeliczyc, bo na buttonie `Start simulation with x number of nodes` jest zle